<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_profile</class>
 <widget class="QMainWindow" name="info_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Информация о профиле</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #18191c;
font: 600 12pt &quot;Segoe UI&quot;;
color: #fff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1e2023;
border: 2px solid #383838;
border-radius: 5px;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font-weight: 700;
border: none;</string>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="profil_id">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font-weight: 700;
border: none;</string>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="profil_email">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-weight: 700;
border: none;</string>
           </property>
           <property name="text">
            <string>Логин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="profil_login">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font-weight: 700;
border: none;</string>
        </property>
        <property name="text">
         <string>UUID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="profil_uuid">
        <property name="styleSheet">
         <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: 700;
border: none;</string>
        </property>
        <property name="text">
         <string>HWID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="profil_hwid">
        <property name="styleSheet">
         <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
